dev/cf_operator/BUILD.bazel

```diff
@@ -10,7 +10,7 @@ helm_template(
     install_name = "cf-operator",
     namespace = project.cf_operator.namespace,
     set_values = {
-        "operator.watchNamespace": project.namespace,
+        "global.operator.watchNamespace": project.namespace,
```

Comment:
Kind of curious about this… needed to propagate it to a subchart or something? (No need to comment in the code, I think)

Reply:
I don't have the exact context for this change on cf-operator.